config/gulp/website.js

```diff
@@ -10,91 +12,122 @@ var taskBuild = task + ':build';
 var taskServe = task + ':serve';
 var taskDev = task + ':development';
 
-// Remove the .bundle/config file generated by `local-gem`
-//
-gulp.task('local-bundle-clean', function (done) {
+gulp.task('clean-generated-assets', function (done) {
+  return del([
+    'docs/_scss',
```

Comment:
If this is the case, where does style specific to the docs site go? Like styleguide.scss or homepage.scss?